src/test/java/WeekOneExercisesPartTwo.java

```diff
@@ -16,20 +17,18 @@ public void bloodAlcoholLevelForAMale() {
     }
 
     @Test
+    @Disabled
     public void bloodAlcoholLevelForAFemale() {
-        char gender = 'M';
-        int weightInPounds = 185;
-        int ouncesOfAlcoholConsumed = 3;
-        float hoursSinceLastDrink = 2;
+        char gender = 'F';
```

Comment:
We had this set to the wrong value. Also, I changed the other inputs so they are different than the first test.